cfg/validators-example.txt

```diff
@@ -27,9 +25,15 @@
 #
 #   List of URIs serving lists of recommended validators.
 #
+#   The latest list of recommended validator sites and keys can be
```

Comment:
Tweak: `The latest list of recommended validator sites can be`

Reply:
Yeah, I like that. Fixed.